base/shell/explorer/resource.h

```diff
@@ -216,3 +217,5 @@
 #define ID_SHELL_CMD_CUST_NOTIF         (411)
 #define ID_SHELL_CMD_ADJUST_DAT         (412)
 #define ID_SHELL_CMD_RESTORE_ALL        (413)
+#define ID_SHELL_CMD_MINIMIZE_GRP       (414)
+#define ID_SHELL_CMD_CLOSE_GRP          (415)
```

Comment:
If PR #6872 is merged before this (likely) then you need to adjust ID_SHELL_CMD_LAST before merging (likely a conflict).

Reply:
Yeah i know, currently it's a TODO PR

Reply:
The only thing is that ID_SHELL_CMD_LAST was 0x7FEF so it should be fine

Reply:
It was yes but not after my PR is merged.

Reply:
Ah, then totally true